SEPADebit/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
This file gets auto generated. We can remove since this is covered by the top-level `.gitignore`.